lib/web_ui/lib/src/engine/view_embedder/flutter_view_manager.dart

```diff
@@ -93,6 +96,12 @@ class FlutterViewManager {
     return _jsViewOptions[viewId];
   }
 
+  void _init() {
+    // Adds the focus tracking event listeners.
+    domDocument.body?.addEventListener(_focusin, _focusChangeHandler, true);
+    domDocument.body?.addEventListener(_focusout, _focusChangeHandler, true);
```

Comment:
Also, I thought `focusin` was enough. Why do we need to listen to both `focusin` and `focusout`?

Reply:
Yeah, realized `element.unfocus()` doesn't trigger any `focusin` event even tho it moves the focus to the body.